# RRTPlanner:
<<<<<<< HEAD
visualize_potential: false
allow_unknown: true
planner_window_x: 0.0
planner_window_y: 0.0
default_tolerance: 0.1
step_size_: 0.25
delta_: 1.0
=======
  # visualize_potential: false
  # allow_unknown: true
  # planner_window_x: 0.0
  # planner_window_y: 0.0
  # default_tolerance: 0.0
step_size_: 0.15
delta_: 0.5
>>>>>>> ff6eabcc20a638faa2459106e5fa0ef3ac5ef42e
run_time_: 5
max_iterations_: 200000
goal_radius_: 1
  
